modules/icon/react/lib/Svg.tsx

```diff
@@ -28,17 +28,24 @@ export default class Svg extends React.Component<SvgProps> {
       return null;
     }
 
-    const assetStyles = [{display: 'inline-block', '& svg': {display: 'block'}}, styles];
-    const assetStyle = css(assetStyles);
-
     return (
-      <span
-        {...elemProps}
-        dangerouslySetInnerHTML={{__html: src.svg}}
-        // Need to combine iconStyle with the className prop, otherwise we'll clobber it
-        // (we'll need to do something like this for each HTML <span> prop we explicitly set in this component)
-        className={cx(assetStyle, elemProps.className)}
-      />
+      <ClassNames>
```

Comment:
This change makes it so that `shallow` in the unit tests no longer works for `Svg` and `Icon`.

`Svg` works if you change it to `mount`, but then you're checking the props rather than the resulting element's className. Needs some finessing.

Haven't been able to get the `toHaveStyleRule` check in `Icon.spec.tsx` to find the width and height with this `ClassNames` wrapper.